Resources/Locale/en-US/escape-menu/ui/options-menu.ftl

```diff
@@ -37,6 +37,7 @@ ui-options-lobby-music = Lobby & Round-end Music
 ui-options-restart-sounds = Round Restart Sounds
 ui-options-event-music = Event Music
 ui-options-admin-sounds = Play Admin Sounds
+ui-options-bwoink-sound = Play AHelp bwoink Sound
```

Comment:
Don't literally refer to the sound as "bwoink", it should be referred to as "notification" or something.